### PR TITLE
Free response

### PR DESCRIPTION
**Pull request by bethshook**

### CODE REVIEW

File: src/components/FreeResponseInput.tsx [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+  color: ${colors.palette.neutralDark};
+  ${props => props.isErrored && TextAreaErrorStyle};
+  ${props => props.showWarning && css`
+    border: 2px solid ${colors.palette.danger};
+  `}
+  background-color: ${props => props.readOnly && colors.palette.neutralCool};
+`;
+TextArea.displayName = 'TextArea';
+
+export const FreeResponseInput = (props: FreeResponseProps) => {
```
Comment (bethshook) [inline src/components/FreeResponseInput.tsx:69]: @jivey I'll incorporate the `Content` component when I'm back on Monday. Feel free to leave any other comments/suggestions.

File: src/components/FreeResponseInput.tsx [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+      <div className="step-card-body">
+          <TextArea
+              {...props}
+              value={response}
+              onChange={setResponse}
+              data-test-id="free-response-box"
+              placeholder="Enter your response..."
+              aria-label="question response text box"
+          />
+          <InfoRow hasSubmitted={!!lastSubmitted}>
```
Comment (bethshook) [inline src/components/FreeResponseInput.tsx:110]: @jivey The first div here seems only relevant to tutor but the second seems like something we want. So I'm curious what you think the best way to handle it is.

File: package.json [hunk 3]
```diff
@@ -28,7 +28,7 @@
     "@types/react": "^16.14.0",
     "@types/react-dom": "^16.9.16",
     "@types/react-test-renderer": "^16.9.5",
-    "@types/styled-components": "^5.1.25",
+    "@types/styled-components": "^5.1.7",
```
Comment (jivey) [inline package.json:31]: Just curious if we need this version for something specific?

File: src/components/FreeResponseInput.tsx (hunk 2 above)
Comment (jivey) [reply src/components/FreeResponseInput.tsx:110]: If it's only to match the Tutor render that seems fine to me, though I'm starting to think this is a component that will diverge anyway

File: src/utils.ts [hunk 4]
```diff
@@ -1,4 +1,5 @@
 import { Answer, ChosenAnswer, ID } from '../src/types';
+import { isString, trimStart, trimEnd } from 'lodash';
```
Comment (jivey) [inline src/utils.ts:3]: I pulled in lodash initially to make the transition easier, but in my components I've been attempting to replace it with built in methods ([this](https://gist.github.com/Kris-CDA/9393c6b0c61185a4c4d9426d5dd3a15e) has been a helpful guide.) If we end up having to need it that's okay, but I like the idea of making the bundle size smaller for the consuming application.

**Review by jivey — APPROVED** [review]

**Pull request merged**